assets/content/cookbook/Beginner/Haxe-To-Emscripten-Hello-World.md

```diff
@@ -0,0 +1,65 @@
+[tags]: / "Haxe, Emscripten"
+# Haxe to Emscripten
+Simple hello world  Haxe code. Then compile and linked to emscrpten.
+But, first I would like to thanks everyone from the Haxe community for helping
+resolving this.
+
+### Procedure
+For some odd reason for this to work. It need both copies of emsdk-1.35.0-full-64bit.msi and emsdk.zip 64bit version.
+
+
+* OS: Windows 7 x64
+* Haxe: V4.x
+* Emscripten: V1.35 and emsdk lastest version
+
+  * Install emsdk-1.35.0-full-64bit.msi (Do not update it or install latest version)
+  * Install emsdk.zip (https://emscripten.org/docs/getting_started/downloads.html)
```

Comment:
Wrap link in `<https://emscripten.org/docs/getting_started/downloads.html>` to make it render correctly.

Reply:
Thanks for your comments. I've updated changes as suggested